app/frontend/styles/application.scss

```diff
@@ -40,6 +40,7 @@ $moj-images-path: "~@ministryofjustice/frontend/moj/assets/images/";
 @import "_application-card";
 @import "_autocomplete";
 @import "_banner";
+@import "_notification_banner";
```

Comment:
Is the intention of this to replace the banner styles (or at least the similar ones) or are we intending of keeping both?

Reply:
We need to keep both as we currently use banners to render form details and the error summary (which the GDS guidelines specifically indicate that should not be replaced by this component). I did consider renaming it, however we already have `summary` components in the app, so I thought it's probably best to leave as it is to avoid confusion.

Reply:
We still need the ordinary banner as we use it to display [form related information](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/views/provider_interface/interviews/new.html.erb#L18).